Makefile.am

```diff
@@ -299,9 +300,10 @@ cpp_tools_ct_clustertool_LDADD = \
 	$(evhtp_LIBS) \
 	$(json_c_LIBS) \
 	$(libevent_LIBS) \
-	-lprotobuf -lsqlite3
+	-lprotobuf -lsqlite3 -ljson-c
```

Comment:
I have to admin my brain had shutdown with the repetitive c'n'p by this point.
Removed.